hello-world.json

```diff
@@ -0,0 +1,33 @@
+{
+   "#": [
+     "Language implementations vary on the issue of missing input variables.",
+     "In this set of test cases, the input variable is left out of the definition,",
+     "and the generator would need to handle the language implementation."
+   ],
+   "cases": [
+      {
+         "description": "no name",
+         "expected": "Hello, World!",
+         "msg": ["When given no name, it should greet the world!"]
```

Comment:
The existing tests contain multi-line commentary. It was simpler to implement this in the comments rather than introduce complexity trying to split long comments into multiple lines intelligently. In addition, since this was the first problem, I felt it was appropriate that the longer comments be preserved as is, rather than shortened.